cmake/CodeCoverage.cmake

```diff
@@ -148,7 +148,6 @@ FUNCTION(SETUP_TARGET_FOR_COVERAGE _targetname _testrunner _outputname)
 		COMMAND ${LCOV_PATH} --directory . --capture --output-file ${coverage_info}
 		COMMAND ${LCOV_PATH} --remove ${coverage_info} 'tests/*' '/usr/*' --output-file ${coverage_cleaned}
 		COMMAND ${GENHTML_PATH} -o ${_outputname} ${coverage_cleaned}
-		COMMAND ${CMAKE_COMMAND} -E remove ${coverage_info} ${coverage_cleaned}
```

Comment:
How does this help? I just would like to learn.

Reply:
This step deletes the coverage output files, which can't be deleted until we upload to Codecov in the next step (after running `CodeCoverage.cmake`). Once we have the upload step here, I think we can restore this clean-up step.